apps/sensenet/src/components/dialogs/reference-content-list/reference-list.tsx

```diff
@@ -41,19 +41,18 @@ export const ReferenceList: React.FC<ReferenceListProps> = (props) => {
   const [requestClearToken, setRequestClearToken] = useState<number>()
   const pagination = usePagination({ items: references, color: 'primary' })
 
-  const fieldControl = createElement(
-    controlMapper.getControlForContentField(props.parent.Type, props.fieldName, 'new'),
-    {
-      repository,
-      settings: field!.fieldSettings,
-      content: props.parent,
-      actionName: 'new',
-      fieldOnChange: (_, value: GenericContent) => {
-        setNewReference(value)
-      },
-      triggerClear: requestClearToken,
-    },
-  )
+  const fieldControl = field
+    ? createElement(controlMapper.getControlForContentField(props.parent.Type, props.fieldName, 'edit'), {
+        repository,
+        settings: field.fieldSettings,
+        content: props.parent,
+        actionName: 'new',
```

Comment:
should it match with the 3rd parameter of getControlForContentField? if yes, which is the correct?